contracts/ICertificate.sol

```diff
@@ -9,4 +9,16 @@ interface ICertificate {
     uint256[] removalIds,
     uint256[] removalAmounts
   );
+
+  /**
+   * @notice Emitted when a removal is released from a Certificate.
+   * @param certificatedId The certificate to connected to the removal.
+   * @param removalId The removal to update the balance for.
+   * @param amount The amount to added or removed for the certificate.
```

Comment:
```suggestion
   * @param amount The amount removed from the certificate.
```

Reply:
releasing is only ever a decrementing of balance.  in the future we have to implement the replacement mechanism which will add new removal tokens to the certificate